x/profiles/client/cli/cli_profile.go

```diff
@@ -96,7 +97,9 @@ func GetCmdQueryProfile() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "profile [address_or_dtag]",
 		Short: "Retrieve the profile having the specified user address or profile dtag, if any.",
-		Args:  cobra.ExactArgs(1),
+		Example: fmt.Sprintf(`%s query profiles desmos12a2y7fflz6g4e5gn0mh0n9dkrzllj0q5vx7c6t
+%s query profiles leoDiCaprio`, version.AppName, version.AppName),
```

Comment:
I would use standard names here (Alice, Bob, etc)
```suggestion
%s query profiles Alice`, version.AppName, version.AppName),
```